org.ghost.desktop.json

```diff
@@ -0,0 +1,40 @@
+{
+  "app-id": "org.ghost.desktop",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.gnome.Sdk",
+  "command": "Ghost",
+  "rename-desktop-file": "Ghost.desktop",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--filesystem=host",
+    "--share=network",
+    "--device=dri",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "modules": [
+    {
+      "name": "ghost",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar -x ghost.deb",
+        "tar -xf data.tar.xz",
+        "cp -r usr/* /app",
+        "mkdir -p /app/share/icons/hicolor/512x512/apps",
+        "cp /app/share/pixmaps/Ghost.png /app/share/icons/hicolor/512x512/apps/org.ghost.desktop.png",
+        "desktop-file-edit --set-key=Icon --set-value='org.ghost.desktop' /app/share/applications/Ghost.desktop"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://github.com/TryGhost/Ghost-Desktop/releases/download/v1.5.2/ghost-desktop-1.5.2-debian.deb",
+          "sha256": "dbd6c98796be97ad9d9e04b1300f8e2beb87a9a0ea0f4358f568a7b2c16a0f6a",
+          "dest-filename": "ghost.deb"
```

Comment:
Does the Debian package work for all the architectures? If not, you might need to specify that.

Reply:
It is the only package, but seems to just be for amd64. Is it enough to just add the arch to the source?